src/Accelerators/Accelerator.cpp

```diff
@@ -25,5 +25,25 @@ const llvm::SmallVectorImpl<Accelerator *> &Accelerator::getAccelerators() {
   return acceleratorTargets;
 }
 
+// Help to print accelerator kinds.
+static std::map<Accelerator::Kind, std::string> mapKind2Strings;
```

Comment:
Should we also `#include <map>` at the top of the file?

Reply:
yes, thanks for noticing it